config/args.js

```diff
@@ -1,7 +1,11 @@
 const commandLineArgs = require('command-line-args');
 
 const scriptNameRegex = /scripts[\/\\]([\w-]*)\.js$/i;
-const scriptName = process.argv[1].match(scriptNameRegex)[1];
+const scriptNameByRegex = process.argv[1].match(scriptNameRegex);
+
+const scriptName = scriptNameByRegex
```

Comment:
I'm thinking we should move to [command-line-commands](https://github.com/75lb/command-line-commands)?